app/models/street/StreetEdgeTable.scala

```diff
@@ -221,22 +369,41 @@ object StreetEdgeTable {
     * @return
     */
   def countAuditedStreets(auditCount: Int = 1): Int = db.withSession { implicit session =>
-    selectAuditedStreets(auditCount).size
+    selectAuditedStreets(auditCount, completedAuditTasks).size
+  }
+
+  def countTurkerAuditedStreets(auditCount: Int = 1): Int = db.withSession { implicit session =>
+    selectAuditedStreets(auditCount, turkerCompletedAuditTasks).size
+  }
+
+  def countRegisteredUserAuditedStreets(auditCount: Int = 1): Int = db.withSession { implicit session =>
+    selectAuditedStreets(auditCount, regUserCompletedAuditTasks).size
+  }
+
+  def countResearcherAuditedStreets(auditCount: Int = 1): Int = db.withSession { implicit session =>
+    selectAuditedStreets(auditCount, researcherCompletedAuditTasks).size
+  }
+
+  def countAnonAuditedStreets(auditCount: Int = 1): Int = db.withSession { implicit session =>
+    selectAuditedStreets(auditCount, anonCompletedAuditTasks).size
   }
 
   /**
     * Returns a list of street edges that are audited at least auditCount times
     *
     * @return
     */
-  def selectAuditedStreets(auditCount: Int = 1): List[StreetEdge] = db.withSession { implicit session =>
+  def selectAuditedStreets(auditCount: Int = 1, auditTasksQuery: Query[AuditTaskTable, AuditTask, Seq] = completedAuditTasks): List[StreetEdge] = db.withSession { implicit session =>
     val edges = for {
-      (_streetEdges, _auditTasks) <- streetEdgesWithoutDeleted.innerJoin(completedAuditTasks).on(_.streetEdgeId === _.streetEdgeId)
+      (_streetEdges, _auditTasks) <- streetEdgesWithoutDeleted.innerJoin(auditTasksQuery).on(_.streetEdgeId === _.streetEdgeId)
     } yield _streetEdges
 
+    //print(edges.groupBy(x => x).map(_._1).list.size)
+    //print(" ")
+
     val uniqueStreetEdges: List[StreetEdge] = (for ((eid, groupedEdges) <- edges.list.groupBy(_.streetEdgeId)) yield {
-      // Filter out group of edges with the size less than the passed `auditCount`
-      if (auditCount > 0 && groupedEdges.size >= auditCount) {
+      // Filter out group of edges with the size not equal to the passed `auditCount`
+      if (auditCount > 0 && groupedEdges.size == auditCount) {
```

Comment:
Updated the audit count comparison to be == instead of >=. However, this change leads to numbers being way below the expected (see screenshot below - `Audited Streets` section). So I am not sure if this code (that filters out streets that don't have the audit count of `auditCount` is working properly. Note: the filtering code is the old code that has been there from the beginning. Only change I made is the equality comparison. 

![image](https://user-images.githubusercontent.com/2873216/31964365-9d8a9a24-b8b8-11e7-8308-c99c64482359.png)


@misaugstad Can you confirm if my observation is correct?

Reply:
This function was always designed to filter out streets that were audited _less than_ `auditCount` times. The idea is that if you pass in a value of 1, you are getting all streets that have been audited at least once.

Not sure why you are changing it to be strict equality? If you do that, then of course the numbers are going to be lower than expected; we are now excluding any street edge that was audited twice.

Reply:
Yes, I think I made a mistake in my understanding. I sort of assumed that if we gave audit count of 1 then it should include all streets that have been audited at least once. Forgot that that value is the total number of times it had been audited. Sorry, stupid mistake!